src/EnergyPlus/BaseboardRadiator.cc

```diff
@@ -1183,12 +1164,7 @@ namespace BaseboardRadiator {
             baseboard->Baseboard(BaseboardNum).WaterOutletEnthalpy = baseboard->Baseboard(BaseboardNum).WaterInletEnthalpy;
             WaterMassFlowRate = 0.0;
 
-            SetActuatedBranchFlowRate(state,
-                                      WaterMassFlowRate,
-                                      baseboard->Baseboard(BaseboardNum).WaterInletNode,
-                                      baseboard->Baseboard(BaseboardNum).LoopNum,
-                                      baseboard->Baseboard(BaseboardNum).LoopSideNum,
-                                      baseboard->Baseboard(BaseboardNum).BranchNum,
+            SetActuatedBranchFlowRate(state,                                      WaterMassFlowRate,                                      baseboard->Baseboard(BaseboardNum).WaterInletNode,                                      baseboard->Baseboard(BaseboardNum).plantLoc,
```

Comment:
Formatting seems to have gotten odd here, assuming this is otherwise ready I'll just apply formatting and push this back up.